static/simple.css

```diff
@@ -1,44 +1,40 @@
-
-.container{
+.container {
```

Comment:
Re-organinzing your css code would make your page better. I might be a good help to use some useful tools like bootstrap.